packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIconWrapperStyles.ts

```diff
@@ -25,6 +25,7 @@ const getCheckboxIconWrapperStyles = ({
   const borderColor = border === 'transparent' ? border : getIn(theme, border);
 
   return {
+    position: 'relative',
```

Comment:
can you add a story as well for a lot of checkboxes(a good one not lorem ipsum types 😄)?

Reply:
> lot of checkboxes(a good one not lorem ipsum types 😄)?

If there are lot of checkboxes will be better to suggest people to use MultiSelect in that case. (You'll need like 40 or 50 checkboxes to reproduce this bug on page level 😅)

Also, I don't think we should add stories for fixed bugs. Maybe can add it in tests but then again this bug won't be caught in there.

Reply:
> I don't think we should add stories for fixed bugs.
we shall because to prevent future regressions, ideally should be part of tests but since this can't be part of tests we need to put it in stories.

Reply:
> maybe add a block of say 30-40px height and add 6-7 checkboxes

Sure, can do that. but fyi still won't catch the regression since you need to scroll the overflow container to see this bug. 

(Good usecase for the storybook play functions though :p)

Reply:
Added in https://github.com/razorpay/blade/pull/982/commits/ab15b812c998937f4ea9e49f7b63d1aa741eee9c